From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Isaac J. Manjarres" <isaacm@codeaurora.org>
Date: Tue, 10 Sep 2019 15:42:05 -0700
Subject: FROMLIST: of: Export of_phandle_iterator_args() to modules

Kernel modules may want to use of_phandle_iterator_args(),
so export it.

Change-Id: Ic42b0917834669dd4dc6930081581d66381c7d5a
Signed-off-by: Isaac J. Manjarres <isaacm@codeaurora.org>
Acked-by: Rob Herring <robh@kernel.org>
Bug: 140290589
Link: https://patchwork.kernel.org/patch/10948415/
---
 drivers/of/base.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/of/base.c b/drivers/of/base.c
index 1d667eb730e1..ff896fcc8efa 100644
--- a/drivers/of/base.c
+++ b/drivers/of/base.c
@@ -1404,6 +1404,7 @@ int of_phandle_iterator_args(struct of_phandle_iterator *it,
 
 	return count;
 }
+EXPORT_SYMBOL_GPL(of_phandle_iterator_args);
 
 static int __of_parse_phandle_with_args(const struct device_node *np,
 					const char *list_name,
